src/tools/publish_toolstate.py

```diff
@@ -17,13 +17,22 @@
 import copy
 import datetime
 import collections
+import textwrap
+try:
+    import urllib2
+except ImportError:
+    import urllib.request as urllib2
 
 # List of people to ping when the status of a tool changed.
 MAINTAINERS = {
     'miri': '@oli-obk @RalfJung @eddyb',
     'clippy-driver': '@Manishearth @llogiq @mcarton @oli-obk',
     'rls': '@nrc',
     'rustfmt': '@nrc',
+    'book': '@carols10cents @steveklabnik',
+    'nomicon': '@frewsxcv @Gankro',
+    'reference': '@steveklabnik @Havvy',
+    'rust-by-example': '@steveklabnik',
```

Comment:
↑ I just fill in these names from the recent contributors. Please suggest an actual list.

Reply:
I'm less sure about the nomicon (@Gankro that's primarily your baby) but for the Reference we have @havvy, @matthewjasper, and @alercah as official peers, and for RBE we have @marioidival and @projektir as official peers. @rust-lang/docs Anyone i'm missing?